src/theme.ts

```diff
@@ -5,7 +5,13 @@ import { Context } from '@graasp/sdk';
 
 export const PRIMARY_COLOR = '#5050d2';
 export const SECONDARY_COLOR = '#FFFFFF';
+export const DEFAULT_TEXT_PRIMARY_COLOR = '#313131';
+export const DEFAULT_TEXT_SECONDARY_COLOR = '#71717A';
+export const DEFAULT_ACTIVE_ACTION_COLOR = '#777578';
+export const DEFAULT_LIGHT_PRIMARY_COLOR = '#E4DFFF';
```

Comment:
Here I explicitly set primary and secondary text colors that do not use transparency, as this is the most effective way of making the lucid icons look good. As they have overlaping paths, colors with opacities make these show.

Reply:
add in comments.